tests/modules/core/lib/Auth/Process/PHPTest.php

```diff
@@ -32,10 +33,8 @@ private static function processFilter(array $config, array $request)
     public function testInvalidConfiguration()
     {
         $config = [];
-        $this->expectException(
-            \SimpleSAML\Error\Exception::class,
-            "core:PHP: missing mandatory configuration option 'code'."
-        );
+        $this->expectException(\SimpleSAML\Error\Exception::class);
+        $this->expectExceptionMessage("core:PHP: missing mandatory configuration option 'code'.");
         new \SimpleSAML\Module\core\Auth\Process\PHP($config, null);
```

Comment:
Not sure in what PHPunit-version this was allowed, but today it's split in two calls (actually for years already)